deployment/helm/node-feature-discovery/templates/master.yaml

```diff
@@ -6,8 +6,10 @@ metadata:
   labels:
     {{- include "node-feature-discovery.labels" . | nindent 4 }}
     role: master
+  {{- with .Values.master.deploymentAnnotations }}
   annotations:
-    {{- toYaml .Values.master.deploymentAnnotations | nindent 4 }}
+    {{- toYaml . | nindent 4 }}
```

Comment:
I'm not a Helm expert but why not use the simpler oneliner format used elsewhere in this chart?

```suggestion
    {{- toYaml .Values.master.deploymentAnnotations | nindent 8 }}
```

Reply:
@andrewjamesbrown thank you for the explanation. Makes sense. As said my Helm-fu is limited 😇 As a bit of a sidenote,  do you think that some other fields (like `labels`) should be handled similarly.

Reply:
As far as I can tell, labels are always set through values/helpers, so they'll always be present in the default case.  Which means there's no reason to conditionally add them using `with`

Reply:
K, thanks for educating me